common/djangoapps/student/models.py

```diff
@@ -1442,6 +1442,28 @@ def enrollments_for_user(cls, user):
         return cls.objects.filter(user=user, is_active=1)
 
     @classmethod
+    def preloaded_enrollments_for_user(cls, user):
+        """
```

Comment:
I wonder if the interface for this feature could be simplified as an optional keyword argument to `enrollments_for_user`, e.g.
```
def enrollments_for_user(cls, user, preload_course_overviews=False):
```